docs/build_conf.md

```diff
@@ -103,6 +103,7 @@ RETRY=false           # Error Recovery disabled
 Option to get device serial from system BIOS table:
 GET_DEV_SERIAL=true            # get device serial enabled
 GET_DEV_SERIAL=false           # get device serial disabled (default)
+NOTE: linux-client required elevated privileges to get device serial from system BIOS table. Use 'sudo' to execute.
```

Comment:
dmidecode -s system-serial-number is a linux command and it requires permissions to display system details.
Sudo is required to get a serial number from BIOS system. If sudo is not provided then it will take the default 'abcdef' serial number